docs/index.md

```diff
@@ -1,36 +1,62 @@
 ---
 layout: ""
-page_title: "Provider: kentik-cloudexport"
+page_title: "Provider: Kentik CloudExport"
 description: |-
-  The kentik-cloudexport provider provides resources to interact with Kentik CloudExport API.
+  The Kentik CloudExport provider provides resources to interact with Kentik CloudExport API.
 ---
 
-# kentik-cloudexport provider
+# Kentik CloudExport Provider
 
-The kentik-cloudexport provider provides resources to interact with Kentik CloudExport API.
+The Kentik CloudExport provider provides resources to interact with Kentik CloudExport API.
```

Comment:
A better version:

"The kentik-cloudexport provider allows to interact with the Kentik Cloud Export API"

We refer to the cloud export functionality and related API with a title-case string (see the [API Tester)](https://portal.kentik.com/v4/api-tester/cloud_export-202101beta1/swagger).

Reply:
Done